docs/src/sqlitegen/bubble-generator-data.tcl

```diff
@@ -491,13 +491,21 @@ set all_graphs {
       }
   }
 
+
+
   constraint-section {
       loop
       {stack
-          {line /keyname -> < /ref-table-name : /ref-keyname > }
-          {line
-              {opt on update cascade}
-              {opt on delete cascade}
+          {line /keyname -> 
+               {or 
+                    {line /ref-table-name : /ref-keyname }
+                    {line {loop {line < /ref-table-name : /ref-keyname > } } }
+               }
+          }
+          {opt 
+            {loop 
+               {line on {or update delete} {or {line cascade} {line restrict} }}
```

Comment:
Please use upper-case letters for keywords.

Reply:
these keywords seem to be case sensitive, not sure if we want to confuse by putting in upper case.